cli/src/commit_templater.rs

```diff
@@ -2264,3 +2333,25 @@ pub fn builtin_annotation_line_methods<'repo>(
     );
     map
 }
+
+pub fn builtin_trailer_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, Trailer> {
```

Comment:
nit: remove `pub` (appears that some existing functions unintentionally have `pub`)

Reply:
should I remove the unnecessary `pub`s from the other functions?

Reply:
https://github.com/jj-vcs/jj/pull/6420